### ISSUE TITLE
Crash on user login when WSU API returns no data

### ISSUE THREAD

**Issue by admturner** [washingtonstateuniversity/WSUWP-Plugin-A11y-Status#56]
## Description

Error thrown when the Accessibility Status plugin tries to refresh a user's status and doesn't receive data from the external API.

## Screenshots or screen recording (optional)

~~~
Error Details
=============
An error of type E_ERROR was caused in line 117 of the file /var/www/html/wp-content/plugins/wsuwp-plugin-a11y-status/includes/class-wsu-api.php. Error message: Uncaught TypeError: array_shift(): Argument #1 ($array) must be of type array, null given in /var/www/html/wp-content/plugins/wsuwp-plugin-a11y-status/includes/class-wsu-api.php:117
Stack trace:
#0 /var/www/html/wp-content/plugins/wsuwp-plugin-a11y-status/includes/class-wsu-api.php(117): array_shift(NULL)
#1 /var/www/html/wp-content/plugins/wsuwp-plugin-a11y-status/includes/class-wsu-api.php(73): WSUWP\A11yStatus\WSU_API\WSU_API->fetch_api_response()
#2 /var/www/html/wp-content/plugins/wsuwp-plugin-a11y-status/includes/user.php(94): WSUWP\A11yStatus\WSU_API\WSU_API->__construct('URL', 'NID')
#3 /var/www/html/wp-content/plugins/wsuwp-plugin-a11y-status/includes/user.php(47): WSUWP\A11yStatus\user\update_a11y_user_meta(Object(WP_User))
#4 /var/www/html/wp-includes/class-wp-hook.php(308): WSUWP\A11yStatus\user\handle_user_login('NID', Object(WP_User))
#5 /var/www/html/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#6 /var/www/html/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /var/www/html/wp-includes/user.php(121): do_action('wp_login', 'NID', Object(WP_User))
#8 /var/www/html/wp-login.php(1241): wp_signon(Array, true)
#9 {main}
  thrown
~~~

## WordPress information
- WordPress version: 6.2.2
- Are all plugins except HRS Blocks deactivated? No
- Are you using the HRS child theme? Yes
